en/documentation/index.md

```diff
@@ -144,9 +149,9 @@ Here is a list of popular tools used by Rubyists:
 
 ### Further Reading
 
-[Ruby-Doc.org][34] maintains a comprehensive list of English
-documentation sources. There are also plenty of [books about Ruby][35].
-If you have questions about Ruby the [mailing
+[Ruby-Doc.org][34] and [Omniref.com][40] both maintain a comprehensive 
```

Comment:
The site has documentation for every version of every RubyGem and every
version of Ruby. There's a list, but most people won't use it that way,
because search is much easier:

https://www.omniref.com/ruby/gems

On Sun, Feb 22, 2015 at 7:25 AM, Marcus Stollsteimer <
notifications@github.com> wrote:

> In en/documentation/index.md
> <https://github.com/ruby/www.ruby-lang.org/pull/974#discussion_r25132933>:
>
> > @@ -144,9 +149,9 @@ Here is a list of popular tools used by Rubyists:
> >
> >  ### Further Reading
> >
> > -[Ruby-Doc.org][34] maintains a comprehensive list of English
> > -documentation sources. There are also plenty of [books about Ruby][35].
> > -If you have questions about Ruby the [mailing
> > +[Ruby-Doc.org][34] and [Omniref.com][40] both maintain a comprehensive
>
> @timaro <https://github.com/timaro> I couldn't find such a comprehensive
> list on the Omniref page.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ruby/www.ruby-lang.org/pull/974/files#r25132933>.
>



-- 
Tim Robertson
Founder, https://www.omniref.com
A code-centric universal reference manual, by and for the programming
community.

Check us out on Twitter: https://twitter.com/omniref
Talk about Omniref on our community subreddit:
http://www.reddit.com/r/omniref